lib/fs_utils/write.js

```diff
@@ -162,6 +171,8 @@ const write = (fileList, config, joinConfig, optimizers, startTime) => {
   })).then(() => {
     return writeStatic(fileList, config, startTime);
   }).then(() => {
+    if (config.offlineFirst.enabled) return writeSWPreCache(config);
```

Comment:
Is there a need to wait for `writeStatic` to complete before calling `writeSWPreCache`? Can we do it in parallel?